proto/vschema.proto

```diff
@@ -45,6 +45,15 @@ message Keyspace {
   map<string, Table> tables = 3;
   // If require_explicit_routing is true, vindexes and tables are not added to global routing
   bool require_explicit_routing = 4;
+  // foreign_key_mode dictates how Vitess should handle foreign keys for this keyspace.
+  ForeignKeyMode foreign_key_mode = 5;
+
+  enum ForeignKeyMode {
+    Fk_default = 0;
```

Comment:
Enum value identifiers are mostly uppercase with underscores, though people have introduced a few that use CamelCase. Can you change these to be similar to this example? We don't need a third way of naming these.
```
DEFAULT_PLANNER = 0;
```

Reply:
I have changed the case to be all caps-lock 👍